src/routes/api/assets/patchById.js

```diff
@@ -1,16 +1,85 @@
 const logger = require("../../../logger");
-const { createSuccessResponse } = require("../../../response");
+const prisma = require("../../../model/data/prismaClient");
+const {
+  createSuccessResponse,
+  createErrorResponse,
+} = require("../../../response");
 
 /**
- * Update an asset by it's id
+ * Update an asset by its id
  */
-
-// eslint-disable-next-line no-unused-vars
 module.exports = async (req, res, next) => {
-  logger.debug(
-    { user: req.user, id: req.params.id },
-    `received request: PATCH /v1/assets/:assetId`,
-  );
+  try {
```

Comment:
suggestion: We could scope down this try block to separate try blocks which wrap around the code that throws.

This will improve readability by reducing nesting and make debugging easier because we can more easily identify where the error occurred.